process/block/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -37,57 +38,46 @@ type ArgsAlteredAccountsProvider struct {
 }
 
 type alteredAccountsProvider struct {
-	shardCoordinator            sharding.Coordinator
-	addressConverter            core.PubkeyConverter
-	accountsDB                  state.AccountsAdapter
-	marshalizer                 marshal.Marshalizer
-	fungibleTokensIdentifier    map[string]struct{}
-	nonFungibleTokensIdentifier map[string]struct{}
+	shardCoordinator   sharding.Coordinator
+	addressConverter   core.PubkeyConverter
+	accountsDB         state.AccountsAdapter
+	marshalizer        marshal.Marshalizer
```

Comment:
Perhaps `marshaller` ?

Reply:
The interface is named `Marshalizer` so I just kept the naming for consistency. Should be renamed everywhere